proposals/revised-governance.md

```diff
@@ -0,0 +1,227 @@
+ 
+## tl;dr version
+
+This proposal eliminates the TSC as a standing body of individual voting
+members and instead distributes the decision making authority previously
+held by that standing body to the chartered working groups.
+
+This proposal does not change the CTC in any way other than in terms of how
+new working groups are chartered.
+
+## Proposal
+
+What I am proposing is a revised TSC structure such that:
+
+1. Eliminate the TSC standing membership body. Rather than individual voting
+   TSC members, each chartered working group has a single voting seat. It is
+   up to each working group to decide on its own process for exercising that
+   vote.
+2. An Admin team would be created that would be charged with handling the
+   more mundane administrative tasks currently handled by the TSC (budget
+   allocations, github org management, handling confidential information).
+   Tasks like administering the travel fund could be handed off to the
+   Community Committee. This admin team would not be a voting body.
+3. There would be only one TSC meeting per month.
+   * The meeting may be in person or fully remote.
+   * The meeting would consist of a public portion only that includes a
+     stand up of *working group* activities and issues, and discussion of
+     escalated issues.
+   * Any working group member may attend.
+   * Anyone who is not a working group member may attend on invitation from a
+     working group member.
+4. The TSC Chair and TSC Director positions remain with an annual election.
+   * Any working group member is eligible to run for either seat and all working
+     group members would vote.
+   * The TSC Chair organizes the agenda for, and conducts, the monthly meeting.
+   * The TSC Director sits on the Node.js Board of Directors and chairs the
+     Admin team.
+5. Any working group member may request to participate in the Admin team, and
+   the Admin team may have a process for adding non-working group members.
+   All Admin team members are subject to consensus or vote by the working
+   groups. 
+6. The CTC does not change with the exception that it does not charter its own
+   working groups.
+7. Working groups would be chartered through consensus of all the working
+   groups, including the CTC. The CTC would not have sole ability to override
+   a working group decision. If the CTC objected to a decision made by 
+   chartered working group, the final decision would be made through consensus
+   or vote of all the working groups. 
+
+## Questions
+
+### Why not simply fold the TSC and CTC back together?
+
+For the simple reason that doing so does not address the cause of the
+disfunction.
+
+Also, the reason the TSC and CTC were split in the first place was because the
+overwhelming majority of CTC members do not want to deal with the administrative
+tasks delegated to the TSC, and I imagine that is still the case. We would still
+need an Admin team to handle those tasks.
+
```

Comment:
I don't believe that would work entirely because the Admin team would, in fact, also be handling matters that are specific to the Node.js project. For instance, adding or removing people from teams, repository management, team management, etc... generally any of the GitHub owner role tasks. Also, the Admin team would essentially serve as the go between for the Foundation and this new TSC. On the rare occasion confidential details needed to be shared with the project, for instance --- which has happened from time to time. 

This is not to say that some responsibilities could not be shifted. For instance, administration of the travel fund could easily move over to CommComm, as could issues around Code of Conduct and moderation. (I would like to see the CommComm come up with a moderation policy before we switched that over entirely tho). The point, however, is that there will always be a certain amount of administrative overhead in Core itself that cannot be delegated.